model/dish.go

```diff
@@ -0,0 +1,13 @@
+package model
+
+import "time"
+
+type Dish struct {
+	DishId int
+	RestId int
+	Price float64
+	DishName string
+	Stock int
+	Sales int
+	CreatTime time.Time
+}
```

Comment:
please add one more line at EOF(end of file)

Reply:
thanks, updated.